README.md

```diff
@@ -1240,6 +1240,176 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='blank-line-after-multiline-switch-case'></a>(<a href='#blank-line-after-multiline-switch-case'>link</a>) **Insert a blank line following a switch case with a multi-line body.** Spacing within an individual switch statement should be consistent. If any case has a multi-line body then all cases should include a trailing blank line. The last switch case doesn't need a blank line, since it is already followed by a closing brace. [![SwiftFormat: blankLineAfterMultilineSwitchCase](https://img.shields.io/badge/SwiftFormat-blankLineAfterMultilineSwitchCase-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#blankLineAfterMultilineSwitchCase) [![SwiftFormat: consistentSwitchStatementSpacing](https://img.shields.io/badge/SwiftFormat-consistentSwitchStatementSpacing-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#consistentSwitchStatementSpacing)
+
+  <details>
+
+  #### Why?
+
+  Like with [declarations in a file](https://github.com/airbnb/swift#newline-between-scope-siblings), inserting a blank line between scopes makes them easier to visually differentiate.
+  
+  Switch statements with multi-lines lines can become visiually busy and cramped without blank lines between the cases. In this example, all of the cases start to bleed together, making it more difficult to read the code at a glance:
```

Comment:
In case it's helpful, I think another way that we can frame the justification is that the newlines make it easier to read more complex switch statements. When a switch statement contains code that is multiple lines, now the pattern of the switch statement is broken (e.g. case statement followed by one line of code) and so introducing the blank lines makes it easy to differentiate the cases at a glance.

Reply:
I like emphasizing how this style makes it easier to distinguish between individual cases, thanks!